Dockerfile

```diff
@@ -1,10 +1,28 @@
+FROM golang:latest AS build
```

Comment:
How large is the resulting container image with this approach?

Reply:
The special part is the separate `FROM alpine:3.6` below - this first image is only used for building, then the result is copied from this image to the separate final image. This technique is newly possible in ~~docker~~ moby 17.05.0-ce

Reply:
Yep, curious about the results.

Reply:
@mreiferson This is multistage-build feature from Docker 17.05+. First image is used for build, second - for distribution.